src/Symfony/Component/HttpClient/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ CHANGELOG
 
  * Add IPv6 support to `NativeHttpClient`
  * Allow using HTTP/3 with the `CurlHttpClient`
+ * Add RFC 9111–based caching support to `CachingHttpClient` (though some features remain partially supported)
```

Comment:
> (though some features remain partially supported)

This is something that may be added to the class itself to me, not in the changelog. Or by rephrasing it like "Add RFC 9111–based caching partial support to `CachingHttpClient`"

Reply:
adressed in https://github.com/symfony/symfony/pull/59576/commits/dd9c6cb31a9bf2d458715ce19d9ec6a95971d2de